stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/endpoint/MessageApi.kt

```diff
@@ -40,22 +41,25 @@ import retrofit2.http.Query
 @AuthenticatedApi
 internal interface MessageApi {
 
+    // REST documentation: https://getstream.io/chat/docs/rest/#messages-sendmessage
```

Comment:
Instead of normal comment, could we use kdoc for that?

Reply:
Updated